app/models/trends.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Trends
+  def self.table_name_prefix
+    'trends_'
+  end
+
+  def self.links
+    @links ||= Trends::Links.new
+  end
+
+  def self.tags
+    @tags ||= Trends::Tags.new
+  end
+
+  def self.calculate
+    [links, tags].each(&:calculate)
+  end
```

Comment:
This may be an unfortunate name for a method, especially for something under `app/models/` as `ApplicationRecord` objects do have a `calculate` method, that does something completely different. Maybe `refresh!`? At least ending with an exclamation mark would make sense, I think.